k8s/nginx-https/container/nginx.conf.template

```diff
@@ -135,7 +147,7 @@ http {
     location / {
         add_header Upgrade "TLS/1.2, HTTP/1.1" always;
         default_type text/plain;
-        return 426 'Consider using the HTTPS protocol next time!';
+        return 301 'Consider using the HTTPS protocol next time!';
```

Comment:
From [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/426):

> The HTTP 426 Upgrade Required client error response code indicates that the server refuses to perform the request using the current protocol but might be willing to do so after the client upgrades to a different protocol.
> The server sends an Upgrade header with this response to indicate the required protocol(s).

Reply:
If you want to use `301`, then maybe add something like:
`return 301 https://$host$request_uri;`